ui/packages/catalog-ui-search/src/main/webapp/component/filter-comparator/filter-comparator.view.js

```diff
@@ -59,6 +61,9 @@ module.exports = Marionette.ItemView.extend({
         if (geometryComparators.indexOf(this.model.get('comparator')) === -1) {
           this.model.set('comparator', geometryComparators[0])
         }
+        if (this.model.get('type') === 'anyGeo') {
```

Comment:
I'm unclear on why these had to be added

Reply:
Ah okay I think I understand now. We added the `any` cases, since `EMPTY` will now be an option for the geo or string cases that _aren't_ `anyText` or `anyGeo` 👍